### DISCUSSION TITLE
Car Info values checkbox

### DISCUSSION THREAD

**Discussion by Pyro1992** [BimmerGestalt/AAIdrive#717] [Ideas]
Don´t know if its possible or not. But my Idea is before we have 20x different SportDisplays with 80% the same stuff, that maybe in the app, that you can choose up to 10 (max?) Values to Display in de Detailedcar settings screen. Then everyone can configure his own Display in the app as he like.
(also in the app as example a checkbox to tick like "Motor oil, Temp, speed.....").

I find some things as steering angle not useful in the SportDisplay section and maybe have some others there to show.

**Comment by hufman** [comment]
With enough development effort, anything is possible, of course! However, I am very bad at UI design, so I am unable to build this. I welcome contributions building such a customization experience, the car protocol is abstracted enough that any good Android developer should be able to figure it out.
